build.sbt

```diff
@@ -27,6 +27,11 @@ scalaVersion := scala212
 crossScalaVersions := supportedScalaVersions
 
 libraryDependencies ++= Seq(
+  if (scalaVersion.value == scala211 || sparkVersion.value == "2.4.2") {
+    "io.delta" %% "delta-core" % "0.6.1"
+  } else {
+    "io.delta" %% "delta-core" % "0.7.0"
```

Comment:
I think we should do "provided" since few cloud providers come with delta JARs built in.

Reply:
Also, let's not worry about 0.7.0 since it is for Spark 3.0